src/ray/raylet_client/raylet_client.cc

```diff
@@ -424,21 +424,45 @@ void raylet::RayletClient::PushMutableObject(
     uint64_t metadata_size,
     void *data,
     const ray::rpc::ClientCallback<ray::rpc::PushMutableObjectReply> &callback) {
-  rpc::PushMutableObjectRequest request;
-  request.set_writer_object_id(writer_object_id.Binary());
-  request.set_data_size(data_size);
-  request.set_metadata_size(metadata_size);
-  // This assumes that the format of the object is a contiguous buffer of (data |
-  // metadata).
-  request.set_data(data, data_size + metadata_size);
-  grpc_client_->PushMutableObject(
-      request,
-      [callback](const Status &status, const rpc::PushMutableObjectReply &reply) {
-        if (!status.ok()) {
-          RAY_LOG(INFO) << "Error pushing mutable object: " << status;
-        }
-        callback(status, reply);
-      });
+  static constexpr uint64_t kMaxGrpcPayloadSize = 1024 * 1024 * 450;  // 450 MiB.
```

Comment:
This is a constant that can be resolved at compile time, so I mark it as a `constexpr` so that the compiler can better optimize the code. `const` is only needed when the value cannot be resolved at compile time.